### ISSUE TITLE
Text Guide should recommend metadata format

### ISSUE THREAD

**Issue by tucotuco** [tdwg/dwc#33]
Was https://code.google.com/p/darwincore/issues/detail?id=180

Reported by gtuco.btuco, May 24, 2013
==Documentation Error Report==
URL:
http://rs.tdwg.org/dwc/terms/guides/text/index.htm

Error description:
Not actually an error, just a request to make a recommendation that isn't there already. The Darwin Core Text Guide does not make recommendations about the metadata document.

Recommended remedy:
Add a recommendation about the metadata in the archive, that it conform minimally to a standard, for example Dublin Core Application Profile.

Submitter: John Wieczorek, Markus Döring

**Comment by tucotuco** [comment]
How to proceed here? The standard is silent on the inclusion of a metadata file describing the data set just as it is on the compression options (see https://github.com/tdwg/dwc/issues/80). That is an innovation of a Darwin Core archive, which is an implementation, documented [here](https://github.com/gbif/ipt/wiki/DwCAHowToGuide).  Should these be part of the standard or only part of the implementation?

**Comment by baskaufs** [comment]
Hmm. There was at one point a desire to maintain "ancillary" pages that contained stuff outside the standard like recommended sources of URIs. I think that is still in the RDF Guide. So I suppose one could create a separate page, perhaps hosted on the DwC GitHub site or the TDWG website, where implementation suggestions are kept. If we did this, we should probably figure out where those RDF ancillary pages ended up and put this in the same place.

**Comment by baskaufs** [comment]
Found the [ancillary page here](https://github.com/tdwg/rdf/blob/master/DwCAncillary.md). It's lurking in the defunct RDF task group's repo, just as it was when exported from Google Code. Probably not the best place for it...

**Comment by tucotuco** [comment]
Wouldn't you put the metadata format recommendation as non-normative
content in the Text Guide, similar to the suggestion for
https://github.com/tdwg/dwc/issues/80?

**Comment by baskaufs** [comment]
I think it could go either way. The whole ancillary page thing dates from before the VMS was written and we were struggling to figure out what the process was for adding a document to a standard. I think the desire to keep that other content out of the standards document was an attempt to have latitude in making changes without invoking a burdensome process. We now have that latitude from the VMS.

In looking at that ancillary page, I'm not sure what is there that needs to be maintained. A lot of it relates to material that may not be so relevant any more. I think the most useful bit there is the list of recommended IRIs, which could probably be put into the RDF Guide as a non-normative section. The other modeling bits could probably be added as references at the end of the RDF Guide. DSW now has a published paper that's citable. The taxon model is being worked on by the new TCS group, so that falls out of the hands of DwC. I'm not aware that the other models discussed are being used by anyone anymore. 

The bottom line is that probably the idea of ancillary pages has outlived its usefulness. This kind of reference material and examples could just be included in the document itself as you suggested.  Also referring people to a page in the repo of a dead task group isn't that great - whatever we maintain should probably be somewhere that can be maintained by the DwC Maintenance Group (another thing that didn't exist prior to the VMS). 

I am so happy at the number of issues that are being closed that I hate to suggest this, but I think I'm going to open an issue to decommission the ancillary RDF page and just revise it into the RDF Guide.

**Comment by tucotuco** [comment]
Punting this issue for later work that also includes the RDF ancillary pag.

**Comment by tucotuco** [comment]
With the [update](4ef3994c65603144748b9c32ffa8a38d2469d3c6) to the Text Guide to mention Darwin Core Archive as an implementation of the Darwin Core Text recommendation and link to the [documentation of Darwin Core Archives](https://github.com/gbif/ipt/wiki/DwCAHowToGuide), the issue of the meta.xml file is taken care of by that implementation documentation, as it should be.
